web/src/components/tasks/business-formation/getErrorStateForField.test.ts

```diff
@@ -1119,7 +1129,7 @@ describe("getErrorStateForField", () => {
   });
 
   describe("fields that have error when undefined", () => {
-    const hasErrorIfUndefined: FormationFields[] = [
+    const onlyHasErrorIfUndefined: FormationFields[] = [
```

Comment:
[pebble] In `getErrorStateForField.ts` we added `paymentType` and `businessSuffix`. Those need to be in this list.

Instead of trying to keep `onlyHasErrorIfUndefined` and `onlyHasErrorIfEmpty` up to date in 2 places, could we export from `getErrorStateForField.ts` and use a single implementation in both places?

Reply:
I agree there should be only list.